cli/exchange/user.go

```diff
@@ -104,13 +121,27 @@ func UserSetHubAdmin(org, userPwCreds, user string, isHubAdmin bool) {
 	var userList ExchangeUsers
 	httpCode := cliutils.ExchangeGet("Exchange", cliutils.GetExchangeUrl(), "orgs/root/users/"+userId, cliutils.OrgAndCreds(org, userPwCreds), []int{200, 404, 403}, &userList)
 	if httpCode == 200 {
-		// If the user already exists in the root org, we need to update it with a PUT to set hubAdmin to correct state
 		exchUser := cliutils.AddOrg("root", userId)
 		if existingUser, ok := userList.Users[exchUser]; !ok {
 			cliutils.Fatal(cliutils.NOT_FOUND, msgPrinter.Sprintf("user '%s' not found in list returned from exchange", exchUser))
 		} else {
-			updatedUser := cliutils.UserExchangeReq{Password: existingUser.Password, Admin: existingUser.Admin, HubAdmin: isHubAdmin, Email: existingUser.Email}
-			cliutils.ExchangePutPost("Exchange", http.MethodPut, cliutils.GetExchangeUrl(), "orgs/root/users/"+userId, cliutils.OrgAndCreds(org, userPwCreds), []int{201}, updatedUser, nil)
+			// emailIsOptional is true (exchange version starting from 2.61.0): 
+			// Hub admin users have to be in the root org. We cannot set isHubAdmin to false for a user in root org
+			// root org doesn't have non-hubadmin user
+
+			// emailIsOptional is false (exchange version before 2.61.0):
+			// Can set hubAdmin to desired boolean value
+			if emailIsOptional, _ := checkExchangeVersionForOptionalUserEmail(org, userPwCreds); emailIsOptional {
```

Comment:
Again, i don't think it is important to check `emailIsOptional` here

Reply:
This is same as above, use `emailIsOptional` to determine user's exchange version

Reply:
This path is a little trickier, because a non-hubadmin user could have existed in the exchange before the exchange stopped allowing it. Maybe we want to pretend that isnt possible and ignore the possibility?